views/partials/roomCard.njk

```diff
@@ -23,6 +23,12 @@
             {% for item in data.acf.prices %}
```

Comment:
Looking at the `ul` classes in this template, `lbh-list` and `lbh-list--bullet` are correct but why is there a `+` in between them? And why `mt0`?

Reply:
Oops. That's a copy paste error on my part from the Typography LBH FE.